include/sway/tree/container.h

```diff
@@ -115,6 +115,9 @@ struct sway_container {
 
 	bool shadow_enabled;
 
+	bool blur_enabled;
+	bool blur_xray;
```

Comment:
what would the use case be for enabling blur_xray per window? I would assume users would want it for just tiled windows, globally, or not at all

Reply:
The xray is always used for tiled windows while this is for floating windows. It creates a cool effect. Would you want it to be a global config-time config option instead?

Reply:
Yeah let's make this global